### PR TITLE
switch "PRIMARY" to "MASTER" on description

### PR DESCRIPTION
**Pull request by lkthomas**
switch "PRIMARY" to "MASTER" on the description; the actual code is correct but the documentation is wrong.

### CODE REVIEW

**Comment by lkthomas** [comment]
Related discussion: https://github.com/ansible-collections/community.mysql/issues/571

File: plugins/modules/mysql_replication.py
Comment (laurent-indermuehle) [inline plugins/modules/mysql_replication.py]:
```suggestion
      C(getreplica) (SHOW REPLICA STATUS),
```

**Review by laurent-indermuehle — CHANGES_REQUESTED** [review]
Thanks for your PR. Good catch on the wrong SQL examples.
I'll merge after you approve my addition of the missing word "STATUS" in the getreplica option. Better fix all exemple in one shot ;)

**Review by laurent-indermuehle — APPROVED** [review]

**Comment by laurent-indermuehle** [comment]
@lkthomas Hi! Yesterday I did something weird, I did a PR on your PR. Don't know if it worked and if you go the notification.
I'm waiting for your reply to merge this PR (after you merged mine)

**Comment by lkthomas** [comment]
no I don't see any notification

**Comment by lkthomas** [comment]
oh hang on, I think I know what you mean, 1 second

**Comment by lkthomas** [comment]
done, check now

**Comment by Andersson007** [comment]
@lkthomas could you please remove the changelog fragment, it's not needed for docs only changes

**Comment by laurent-indermuehle** [comment]
> @lkthomas could you please remove the changelog fragment, it's not needed for docs only changes

oups, my bad. I keep forgetting.

**Comment by lkthomas** [comment]
sorry which changelog fragment? any screenshot?

File: changelogs/fragments/mysql_replicationi_fix_sql_examples.yml
Comment (laurent-indermuehle) [inline changelogs/fragments/mysql_replicationi_fix_sql_examples.yml:1]: This file is the changelog. Sorry I added it, should have known better. Please delete it.

**Review by laurent-indermuehle — CHANGES_REQUESTED** [review]

File: changelogs/fragments/mysql_replicationi_fix_sql_examples.yml
Comment (laurent-indermuehle) [reply changelogs/fragments/mysql_replicationi_fix_sql_examples.yml:1]: @lkthomas I pointed the file that needs to be deleted. Thanks for the work and your contribution!

Comment (Andersson007) [reply changelogs/fragments/mysql_replicationi_fix_sql_examples.yml:1]: @laurent-indermuehle i think you could just delete the file in the WebUI and merge (if tests are green)

Comment (laurent-indermuehle) [reply changelogs/fragments/mysql_replicationi_fix_sql_examples.yml:1]: @Andersson007 I can't do that, Github doesn't let me because the file is in @lkthomas's fork :(

Comment (laurent-indermuehle) [reply changelogs/fragments/mysql_replicationi_fix_sql_examples.yml:1]: Hi @lkthomas, this is a friendly reminder that I'm waiting on your deletion of the fragment file in your fork. When it's done we will be able to merge your contribution.

**Review by Andersson007 — APPROVED** [review]

**Review by laurent-indermuehle — APPROVED** [review]

**Comment by laurent-indermuehle** [comment]
@lkthomas I'm merging your PR now. Thanks again for your contribution!

**Pull request merged**
